<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QLineEdit" name="le_ev0_m">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="QLineEdit" name="le_cc0_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="le_cc0_m">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QLineEdit" name="le_cc0_d">
     <property name="text">
      <string>7</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QLineEdit" name="le_pcd1_d">
     <property name="text">
      <string>15</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="le_cid2_m">
     <property name="text">
      <string>25</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="le_ev1_m">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="le_ev1_d">
     <property name="text">
      <string>6</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLineEdit" name="le_pcd0_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="le_pcd0_m">
     <property name="text">
      <string>25</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="le_pcd0_d">
     <property name="text">
      <string>20</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="le_cid2_d">
     <property name="text">
      <string>15</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="le_cid0_m">
     <property name="text">
      <string>20</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="le_cid0_d">
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLineEdit" name="le_cid2_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLineEdit" name="le_cid1_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLineEdit" name="le_cid3_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="le_cid1_d">
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="le_cid3_d">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_pg0_m">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="le_cid1_m">
     <property name="text">
      <string>20</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="le_cid0_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="le_cid3_m">
     <property name="text">
      <string>20</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="le_pcd1_m">
     <property name="text">
      <string>20</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="15" column="3">
    <widget class="QLineEdit" name="le_pcd1_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="le_ev1_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_pg1_m">
     <property name="text">
      <string>12</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="le_pg0_d">
     <property name="text">
      <string>2</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="le_ev0_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Таможенный контроль</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="9" column="0" rowspan="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="le_ev0_d">
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="le_pg1_d">
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="label_passenger">
     <property name="text">
      <string>Прибытие пассажиров</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>A ± B</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Максимальное
время ожидания</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="le_ev2_m">
     <property name="text">
      <string>10</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="le_ev2_d">
     <property name="text">
      <string>7</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="le_ev2_wt">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Досмотр на входе</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
   <item row="14" column="0" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Паспортный контроль</string>
     </property>
     <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>